### ISSUE TITLE
Duplicated element IDs in mobile version of search results header

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/SearchWorks#3509]
We render two separate versions of this header and its controls — one shown for small & up screens, and one just for xs (mobile) screens. Since a lot of the markup is duplicated, the IDs ended up getting duplicated too.

![Screenshot 2023-09-15 at 15 34 02](https://github.com/sul-dlss/SearchWorks/assets/4924494/8151fe34-a1c9-4236-9fdb-c2e1c1046a59)

The IDs that are duplicated are basically everything except the "next" button:
- `<div id="search-results-toolbar">` (the containing bar itself)
- `<div id="view-type-dropdown">` (the "view" dropdown)
- `<div id="sort-dropdown">` (the sort dropdown)
- `<div id="per_page-dropdown">` (the "per page" dropdown)
- `<button id="select_all-dropdown">` (the "select all" checkbox...oddly named?)

We need to make the IDs for either the mobile version or the non-mobile version different so that they're unique. We probably need to do at least a cursory search to make sure nothing else (e.g. javascript) is looking for those ID values.

Having duplicated IDs on a page is a [WCAG Level A violation](https://www.w3.org/TR/WCAG20-TECHS/F77.html), which is something SODA has asked us to remediate whenever time allows.

[See this issue in SiteImprove](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=3&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

**Comment by cbeer** [comment]
I think we should remove the xs-only navbar. With flexbox (and higher-resolution mobile devices) it probably serves little good.
